.github/workflows/check-cli-reference.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'SpacetimeDB ref'
+        required: false
+        default: ''
+permissions: read-all
+
+name: Check CLI docs
+
+jobs:
+  cli_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find Git ref
+        shell: bash
+        run: |
+          echo "GIT_REF=${{ github.event.inputs.ref || 'master' }}" >>"$GITHUB_ENV"
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          repository: clockworklabs/SpacetimeDB
+          ref: ${{ env.GIT_REF }}
+      - uses: dsherret/rust-toolchain-file@v1
+      - name: Checkout docs
+        uses: actions/checkout@v4
+        with:
+          path: spacetime-docs
+      - name: Check for docs change
+        run: |
+          cargo run --features markdown-docs -p spacetimedb-cli > ../spacetime-docs/docs/cli-reference.md
+          cd spacetime-docs
+          # This is needed because our website doesn't render markdown quite properly.
+          # See the README in spacetime-docs for more details.
+          sed -i'' -E 's!^(##) `(.*)`$!\1 \2!' docs/cli-reference.md
+          sed -i'' -E 's!^(######) \*\*(.*)\*\*$!\1 <b>\2</b>!' docs/cli-reference.md
+          git status
+          if git diff --exit-code HEAD; then
+            echo "No docs changes detected"
+          else
+            echo "It looks like the CLI docs have changed:"
```

Comment:
IMO We should add some extra details here, like if the docs *have* changed then how can they be updated?

Reply:
How's this?